index.js

```diff
@@ -1,116 +1,35 @@
-/* eslint max-classes-per-file: ["error", 2] */
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/228172291-ad746c39-c151-49fd-b63a-ea62f143f67a.png)

**[OPTIONAL]**

- [ ] Kindly remove the highlighted title because you already have a title after Table of Contents 🤔 It would be very great if you remove it and make your readme file more professional and descriptive 😊